cheroot/ssl/builtin.py

```diff
@@ -172,22 +235,155 @@ def get_environ(self, sock):
             'HTTPS': 'on',
             'SSL_PROTOCOL': cipher[1],
             'SSL_CIPHER': cipher[0],
-            # SSL_VERSION_INTERFACE     string  The mod_ssl program version
-            # SSL_VERSION_LIBRARY   string  The OpenSSL program version
+            'SSL_CIPHER_EXPORT': '',
+            'SSL_CIPHER_USEKEYSIZE': cipher[2],
+            'SSL_VERSION_INTERFACE': 'Python %s' % sys.version,
+            'SSL_VERSION_LIBRARY': ssl.OPENSSL_VERSION,
+            'SSL_CLIENT_VERIFY': 'NONE',
         }
 
+        # Python 3.3+
+        if hasattr(sock, 'compression'):
```

Comment:
I think it's cleaner to do
```suggestion
        with contextlib.suppress(AttributeError):
```

Reply:
agreed. changed

Reply:
btw, contextlib.suppress was only introduced in Python 3.4 and not backported to 2.7

Reply:
Interesting. We need to check that we don't use it in the other places, then...